lib/urlwatch/html2txt.py

```diff
@@ -56,25 +61,33 @@ def html2text(data, method='lynx'):
 
     if method == 'pyhtml2text':
         import html2text
-        pyhtml2text = html2text.HTML2Text()
-        d = pyhtml2text.handle(data)
+        parser = html2text.HTML2Text()
+        if len(options) > 0:
+            for o in iter(options):
+                setattr(parser, o.lower(), options[o])
+        d = parser.handle(data)
         return d
 
     if method == 'bs4':
         from bs4 import BeautifulSoup
-        soup = BeautifulSoup(data, 'html.parser')
+        parser = options.pop('parser', 'html.parser')
+        soup = BeautifulSoup(data, parser)
         d = soup.prettify()
         return d
 
     if method == 'lynx':
-        cmd = ['lynx', '-nonumbers', '-dump', '-stdin', '-assume_charset=UTF-8', '-display_charset=UTF-8']
-        stdout_encoding = 'utf-8'
+        cmd = ['lynx', '-nonumbers', '-dump', '-stdin', '-assume_charset UTF-8', '-display_charset UTF-8']
     elif method == 'html2text':
         cmd = ['html2text', '-nobs', '-utf8']
-        stdout_encoding = 'utf-8'
     else:
         raise ValueError('Unknown html2text method: %r' % (method,))
 
+    stdout_encoding = 'utf-8'
+
+    if len(options) > 0:
```

Comment:
There is no point in doing this check, since iterating over an empty dict will do nothing. Also there is no point in using `iter(options)` and then using `options[o]`, instead you can do this and it should be nicer to read and better optimizable by Python:

```
for k, v in options.items():
    cmd.append(...)
```

Also if you want to check if the value is true, do just that (`== True`), not `isinstance(options[o], bool)`, as this will do something unexpected for `{'some_option': False}` (the option will be set anyway).